packages/core/src/components/navigation/NavbarLinks/NavbarLinks.tsx

```diff
@@ -31,16 +37,16 @@ const collections = [
   },
 ]
 
-function NavLinks({ onClickLink, classes = '' }: NavLinksProps) {
+function NavbarLinks({ onClickLink, ...otherProps }: NavbarLinksProps) {
   return (
-    <nav className={`${styles.fsNavlinks} ${classes}`}>
+    <UINavbarLinks {...otherProps}>
       <div className="layout__content">
         <RegionButton />
-        <UIList data-fs-navlinks-list>
+        <UIList data-fs-navbar-links-list>
```

Comment:
Maybe we need to create components for it:
`NavbarLinksList`
`NavbarLinksListItem`

It will follow the same pattern that we used in some Search components. WDYT?